app/src/main/java/memphis/myapplication/utilities/FileManager.java

```diff
@@ -208,7 +210,20 @@ public boolean saveContentToFile(Blob content, Name path) {
                 }
             }
         }
-        byte[] byteContent = content.getImmutableArray();
+
+        String latitude = "";
+        String longitude = "";
+        byte[] byteContent;
+
+        if (location) {
+            byteContent = content.getImmutableArray();
+            latitude = new String(Arrays.copyOfRange(byteContent, byteContent.length - 17, byteContent.length - 9));
+            longitude = new String(Arrays.copyOfRange(byteContent, byteContent.length - 9, byteContent.length));
```

Comment:
Seeing this in action. I don't like it. I think using exif metadata is the best bet here. @agawande ?

Reply:
Replaced this with `androidx.exifinterface.media.ExifInterface`. The coordinates are now stored in the location metadata of the image itself.